plonky2/src/gates/poseidon2_risc0_babybear.rs

```diff
@@ -205,14 +214,14 @@ impl<F: RichField + HasExtension<D>, const D: usize, const NUM_HASH_OUT_ELTS: us
             for i in 0..SPONGE_CAPACITY {
                 let input_lhs = vars.local_wires[Self::wire_input(op, i)];
                 let input_rhs = vars.local_wires[Self::wire_input(op, i + SPONGE_CAPACITY)];
-                let delta_i = vars.local_wires[self.wire_delta(op, i)];
+                let delta_i = vars.local_wires[Self::wire_delta(self.num_ops, op, i)];
```

Comment:
I see that the refactoring of `Poseidon2R0BabyBearGate` made the function calls aligned with the rest of the codebase, but I also noted this "ugly" side-effect that the usage of `wire_delta()` passed from

`self.wire_delta(...)`

to

`Self.wire_delta(self.num_ops, ...)`

Are we sure that static functions are the best option here?